test/e2e/e2e_test.go

```diff
@@ -257,35 +256,35 @@ func createMultipleSignups(t *testing.T, ctx *framework.TestCtx, awaitility *wai
 		userSignup := newUserSignup(t, awaitility.Host(), fmt.Sprintf("multiple-signup-testuser-%d", i), fmt.Sprintf("multiple-signup-testuser-%d@test.com", i))
 		userSignup.Spec.Approved = true
 		err := awaitility.Host().Client.Create(context.TODO(), userSignup, testsupport.CleanupOptions(ctx))
-		awaitility.T.Logf("created UserSignup with username: '%s' and resource name: '%s'", userSignup.Spec.Username, userSignup.Name)
+		awaitility.T.Logf("created usersignupwith username: '%s' and resource name: '%s'", userSignup.Spec.Username, userSignup.Name)
 		require.NoError(t, err)
 		signups[i] = *userSignup
 	}
 	return signups
 }
 
-func verifyMultipleSignups(t *testing.T, awaitility *wait.Awaitility, signups []toolchainv1alpha1.UserSignup, revisions tiers.Revisions) {
+func verifyMultipleSignups(t *testing.T, awaitility *wait.Awaitility, signups []toolchainv1alpha1.UserSignup) {
 	for _, signup := range signups {
-		verifyResourcesProvisionedForSignup(t, awaitility, signup, revisions, "basic")
+		verifyResourcesProvisionedForSignup(t, awaitility, signup, "basic")
 	}
 }
 
-func verifyResourcesProvisionedForSignup(t *testing.T, awaitility *wait.Awaitility, signup toolchainv1alpha1.UserSignup, expectedRevisions tiers.Revisions, tier string) {
+func verifyResourcesProvisionedForSignup(t *testing.T, awaitility *wait.Awaitility, signup toolchainv1alpha1.UserSignup, tier string) {
 	hostAwait := wait.NewHostAwaitility(awaitility)
 	memberAwait := wait.NewMemberAwaitility(awaitility)
-
+	revisions := tiers.GetRevisions(hostAwait, tier)
 	// Get the latest signup version
 	userSignup, err := awaitility.Host().WaitForUserSignup(signup.Name)
 	require.NoError(t, err)
 
 	// First, wait for the MasterUserRecord to exist, no matter what status
-	mur, err := hostAwait.WaitForMasterUserRecord(userSignup.Status.CompliantUsername)
+	mur, err := hostAwait.WaitForMasterUserRecord(userSignup.Status.CompliantUsername, wait.UntilMasterUserRecordHasConditions(provisioned()))
```

Comment:
The comment above explicitly says that we don't care what status of the MUR at this point. The status is checked in line 317: https://github.com/codeready-toolchain/toolchain-e2e/pull/95/files#diff-6413e1dbafd6dd63b5d907ea3ae6f6f9L318

Reply:
ok, I'll revert this change.

Reply:
done in 3e90269